lib/mix/tasks/phx.gen.live.ex

```diff
@@ -0,0 +1,242 @@
+defmodule Mix.Tasks.Phx.Gen.Live do
+  @shortdoc "Generates LiveView, templates, and context for a resource"
+
+  @moduledoc """
```

Comment:
@josevalim I duped most of the `phx.gen.html` docs here, but I wasn't happy with a cross-ref because this may be the first encounter of a user to any phx task and the xref may cause them to miss our notes. Thoughts?

Reply:
:+1: